src/components/read/ReadBible.js

```diff
@@ -80,7 +80,7 @@ const useStyles = makeStyles((theme) => ({
   biblePane: {
     position: "absolute",
     height: "100%",
-    [theme.breakpoints.only("xs")]: {
+    [theme.breakpoints.down("sm")]: {
       width: "100%",
     },
     [theme.breakpoints.up("sm")]: {
```

Comment:
This also needs to change to md here and where ever else its used, about 10 other places
```suggestion
    [theme.breakpoints.up("md")]: {
```